pkg/geo/decode.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Cockroach Authors.
```

Comment:
do you mind renaming this to `geohash.go`?

Reply:
done